core/src/main/java/org/arquillian/smart/testing/configuration/Configuration.java

```diff
@@ -0,0 +1,234 @@
+package org.arquillian.smart.testing.configuration;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import org.arquillian.smart.testing.Logger;
+import org.arquillian.smart.testing.RunMode;
+import org.arquillian.smart.testing.hub.storage.local.LocalStorage;
+import org.arquillian.smart.testing.hub.storage.local.LocalStorageFileAction;
+import org.yaml.snakeyaml.Yaml;
+
+public class Configuration implements ConfigurationSection {
+
+    private static final Logger logger = Logger.getLogger();
+
+    public static final String DEFAULT_MODE = "selecting";
+    public static final String DEFAULT_STRATEGIES = "";
```

Comment:
both variables `DEFAULT_MODE` and `DEFAULT_STRATEGIES` are not used

Reply:
`DEFAULT_MODE` is used in documentation. `DEFAULT_STRATEGIES` are used anywhere

Reply:
then use the `DEFAULT_MODE` also here https://github.com/arquillian/smart-testing/pull/183/files#diff-348861c21efd28f5cc99117b2c67302aR110